make_profiler/lint_makefile.py

```
@@ -48,60 +111,83 @@ def parse_targets(ast: List[Tuple[str, Dict]]) -> Tuple[List[TargetData], Set[st
     return target_data, deps_targets, deps_map
 
 
-def validate_target_comments(targets: List[TargetData], *args, errors: List[str] | None = None) -> bool:
+def validate_target_comments(targets: list[TargetData], *args, errors: list[LintError] | None = None) -> bool:
```

Ruff (0.11.9)

Missing type annotation for `*args`

(ANN002)